.gitignore

```diff
@@ -14,4 +14,5 @@ doc/samples/*.in
 doc/samples/*.out
 doc/samples/file*
 
-
+dev
```

Comment:
Which software uses this?

Usually it is better to set editor ignore files in your user's *gitignore* file. Like `~/.gitignore`

Reply:
`dev` is from doit, in your README you suggest to do 'virtualenv dev'.
`.vscode` is from VisualStudio Code. But putting that one in a personal gitignore sounds like a good idea. Never thought about that. I'll remove that line later.

Reply:
I actually dont do it like that... I use virtualenvwrapper (or more recently virtualfish).